content/hi/docs/reference/glossary/container.md

```diff
@@ -0,0 +1,19 @@
+---
+title: कंटेनर (Container)
+id: container
+date: 2018-04-12
+full_link: /docs/concepts/containers/
+short_description: >
+   एक हल्की और पोर्टेबल एक्सेक्यूटबल योग्य इमेज जिसमें सॉफ़्टवेयर और उसकी सभी निर्भरताएँ शामिल हैं।
```

Comment:
एक्सेक्यूटबल योग्य = निष्पाद्य

Reply:
thanks @Garima-Negi  for suggestions, I will update it asap.